optaplanner-core/src/main/java/org/optaplanner/core/impl/domain/valuerange/descriptor/EntityIndependentValueRangeDescriptor.java

```diff
@@ -25,10 +25,10 @@
 public interface EntityIndependentValueRangeDescriptor<Solution_> extends ValueRangeDescriptor<Solution_> {
 
     /**
-     * As specified by {@link #extractValueRange(Solution_, Object)}.
+     * As specified by {@link ValueRangeDescriptor#extractValueRange}.
```

Comment:
Agreed, I must have confused the old for the new, ignore my original comment, merging. Thanks :)